jenkinsci/jenkinsfile-runner-github-actions
# Updated deprecated 'master' terminology in documentation

This pull request updates the terminology in various files to reflect recent naming changes from "master" to "main". Specifically:

All references to the "master branch" were changed to "main branch" for consistency and accuracy.
Updated Docker commands and file paths that previously referenced "master" to "main" where applicable.
This ensures alignment with modern branch naming conventions and improves readability for new contributors.

Testing Done
Automated Tests: Verified that all automated tests in the repository passed after the changes to confirm no functionality was affected.
Manual Testing:
Ran through the setup instructions using the updated Docker commands to ensure all commands worked as expected.
Checked the GitHub Actions workflows using the modified main branch references, confirming the workflows execute without errors.
Documentation Verification: Scanned the repository to identify any other instances of "master" in documentation and code comments, ensuring complete and consistent updates.
Additional Context
Affected Files:
README.md: Updated references from "master branch" to "main branch."
.github/workflows/...: Updated references in GitHub Actions workflow files to use the main branch.
